prisma/migrations/20231206071528_user_profile/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "UserProfile" ADD COLUMN     "photoUrl" TEXT;
```

Comment:
not important but I think it's better to name it imageUrl or avatarUrl, it can be not only photo but image generated by AI or some art, like character from 'Naruto'